Sources/Widgets/SVG/SVGLandmarkRepresentation/index.js

```diff
@@ -46,7 +49,7 @@ function vtkSVGLandmarkRepresentation(publicAPI, model) {
         const x = xy[0];
         const y = winHeight - xy[1];
 
-        if (model.showCircle === true) {
```

Comment:
@floryst is this notation ok within VTK.JS ?

Reply:
I've changed it to `if (model.circleProps && model.circleProps.visible)`

Reply:
Just as a follow-up: unless you have a strange requirement to verify that a variable is actually the value "true", you don't need the equality check here.